tests/QAN/overview_test.js

```diff
@@ -9,6 +11,27 @@ Before(async ({
   qanFilters.waitForFiltersToLoad();
 });
 
+Scenario(
+  'PMM-T207 Verify hovering over query in overview table  @qan',
+  async ({ I, qanOverview }) => {
+    qanOverview.waitForOverviewLoaded();
+    const firstQueryLocator = `${qanOverview.getRowLocator(1)} div.td:nth-child(2) div`;
+
+    I.waitForVisible(`${firstQueryLocator} > div`, 30);
+    let firstQueryText = await I.grabTextFrom(`${firstQueryLocator} > div`);
+
+    firstQueryText = firstQueryText.replace(/ /g, '');
+
+    I.moveCursorTo(`${firstQueryLocator} > svg`);
+    I.waitForVisible(qanOverview.elements.tooltipQueryValue, 30);
+    I.seeElement(qanOverview.elements.tooltipQueryValue);
+    let tooltipQueryText = await I.grabTextFrom(qanOverview.elements.tooltipQueryValue);
+
+    tooltipQueryText = tooltipQueryText.replace(/ /g, '').replace(/\n/g, '');
+    assert.ok(firstQueryText === tooltipQueryText, 'Expected that the request text in the line matches the request text on the tooltip');
```

Comment:
please improve the assert message to include both expect and actual text, so when the test fails it becomes easy to understand why it does

Reply:
ok